DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/buffered_host_streaming/README.md

```diff
@@ -309,11 +312,79 @@ You should see the following output in the console:
     Average throughput with API: 6370.8013 MB/s
 
     PASSED
-    ```
-    NOTE: In the performance results above, the FPGA kernel is **not** the bottleneck of the full system; the *Producer*/*Consumer* running in parallel are. The full design achieves ~87% of the maximum possible throughput (as measured by the roofline analysis).
+```
+
+## `Buffered Host-Device Streaming` Guided Discussion
```

Comment:
This is the first time that you put the explanations of the sample after the results are shown right?
Do you think that this is a better template?

Reply:
I think in this one case, the tutorial information is better placed below the build instructions. That's the recommendation for other tutorials.